README.md

```diff
@@ -137,11 +137,11 @@ Supported proof assistants:
 Proof General used to support other proof assistants, but those
 instances are no longer maintained nor available in the MELPA package:
 
-* Legacy support of
-  [Isabelle](https://www.cl.cam.ac.uk/research/hvg/Isabelle/) and
+* Experimental support of: Shell
+* Obsolete instances: Demoisa
+* Removed instances: Twelf, CCC, Hol-Light, ACL2, Plastic, Lambda-Clam, HOL98
   [LEGO](http://www.dcs.ed.ac.uk/home/lego)
```

Comment:
Thanks a lot @monnier!

Nitpick in the README.md: one could add two commas here,
```suggestion
* Removed instances: Twelf, CCC, Hol-Light, ACL2, Plastic, Lambda-Clam, HOL98,
  [LEGO](http://www.dcs.ed.ac.uk/home/lego),
```